support-frontend/assets/helpers/productCatalog.ts

```diff
@@ -10,6 +13,37 @@ import { gwDeliverableCountries } from './internationalisation/gwDeliverableCoun
 
 export type { ActiveProductKey };
 
+/*
+ * Ideally, would prefer to loop the ActiveProductKey's generating an ActiveRatePlanKey
+ */
+type OneTimeContributionRatePlanKey = ProductRatePlanKey<'OneTimeContribution'>;
+type GuardianAdLiteRatePlanKey = ProductRatePlanKey<'GuardianAdLite'>;
+type TierThreeRatePlanKey = ProductRatePlanKey<'TierThree'>;
+type DigitalSubscriptionRatePlanKey = ProductRatePlanKey<'DigitalSubscription'>;
+type NationalDeliveryRatePlanKey = ProductRatePlanKey<'NationalDelivery'>;
+type SupporterPlusRatePlanKey = ProductRatePlanKey<'SupporterPlus'>;
+type GuardianWeeklyRestOfWorldRatePlanKey =
+	ProductRatePlanKey<'GuardianWeeklyRestOfWorld'>;
+type SubscriptionCardRatePlanKey = ProductRatePlanKey<'SubscriptionCard'>;
+type ContributionRatePlanKey = ProductRatePlanKey<'Contribution'>;
+type GuardianPatronRatePlanKey = ProductRatePlanKey<'GuardianPatron'>;
+/*
```

Comment:
I don't quite understand this, shouldn't we have GuardianWeeklyDomestic and HomeDelivery rate plans in there as well, even if they have the same names?

Reply:
Presume this relates to further comment below, Updated.